notebooks/templates/due_diligence.ipynb

```diff
@@ -185,13 +185,37 @@
   },
   {
    "cell_type": "code",
-   "execution_count": null,
+   "execution_count": 1,
```

Comment:
I think you accidentally committed an update to the notebook template.

Reply:
Yes I see - I ran a cell to check the mpg package.  I can't figure out how to reset it.

Reply:
You want to copy the file from the repo, overwrite the copy that you have on the branch and commit that.

Reply:
I also found that doing git `checkout [last commit on main branch] -- path_to_file` worked